flocker/node/agents/_logging.py

```diff
@@ -94,3 +99,70 @@
     [VOLUME_ID, STATUS, TARGET_STATUS, WAIT_TIME],
     u"Waiting for a volume to reach target status.",)
 # End: Helper datastructures used by AWS storage driver.
+
+# Begin: Helper datastructures used by OpenStack storage drivers
+
+CODE = Field.for_types("code", [int], u"The HTTP response code.")
+MESSAGE = Field.for_types(
+    "message", [bytes, unicode],
+    u"A human-readable error message given by the response.",
+)
+DETAILS = Field.for_types("details", [dict], u"Extra details about the error.")
+REQUEST_ID = Field.for_types(
+    "request_id", [bytes, unicode],
+    u"The unique identifier assigned by the server for this request.",
```

Comment:
i copied this chunk of code from control/_openstack.py.
In fact, @wallrj pointed out during Boto logging branch review that having both bytes and unicode for message type might not be correct.
My interpretation of having two types was ``gettext`` used by Nova APIs to populate message strings. Since the return type is bytes in Python 2.* and unicode in Python 3.0, we would need both types on our end. - This was my reasoning for what i saw. Please correct me if that is not the case? Thanks!